_includes/search-results.html

```diff
@@ -3,7 +3,7 @@
 <div id="search-result-wrapper" class="d-flex justify-content-center d-none">
   <div class="col-11 content">
     <div id="search-hints">
-      {% include_cached trending-tags.html %}
+      {% include_cached trending-tags.html lang=lang %}
```

Comment:
> ...then this might be due to other modifications in my own version.

Indeed, the issue does not exist in the `production` branch, you can also give it a try.

Additionally, if you want to add parameters here, you should pass the parameters from the upper level, not just `lang`.

```suggestion
      {% include_cached trending-tags.html lang=include.lang %}
```